internal/schema/ize-spec.json

```diff
@@ -37,31 +46,40 @@
                 {
                     "type": "boolean"
                 }
-            ]
+            ],
+            "description": "plain text"
         },
         "prefer_runtime": {
-            "type": "string"
+            "type": "string",
+            "description": "prefer runtime"
         },
         "apps_path": {
-            "type": "string"
+            "type": "string",
+            "description": "path to apps directory"
```

Comment:
```suggestion
            "description": "(optional) Path to apps directory can be set. By default apps are searched in 'apps' and 'projects' directories. This is needed in case your repo structure is not purely ize-structured (let's say you have 'src' repo in your dotnet app, as an example)"
```